docs/emissions.md

```diff
@@ -1,169 +1,111 @@
 ---
-title: "Emissions"
+title: "Emission"
 ---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
-# Emissions
+# Emission
 
-In this document we describe how the Bittensor network, via Yuma Consensus, calculates emissions for a subnet. This calculation is based on the weights that are provided by the root network (subnet 0).
+Emission is the process by which the Bittensor network allocates TAO and alpha to participants, including miners, validators, stakers, and subnet creators.
 
-:::tip Before you proceed
-Read the [Root Network](./root-network.md) document before you proceed.
-:::
+It unfolds in two stages:
 
-## Summary 
+- Injection into subnets
+- Extraction by participants
 
-The emission process works like this:
+See the [Dynamic TAO White Paper](https://drive.google.com/file/d/1vkuxOFPJyUyoY6dQzfIWwZm2_XL3AEOx/view) for a full explanation.
 
-- Every block, i.e., every 12 seconds on the Bittensor blockchain, a single TAO ($\tau$) is minted, i.e., newly created.
-- A percentage portion of this single TAO ($\tau$) is allocated to each subnet in accordance with the subnet's performance. The root network determines the percentage portion for each subnet. Hence, all such partial percentage allocations will sum to 100%, i.e., one TAO ($\tau$). 
-    :::tip Taostats
-    See the [percentage numbers in each "**SN**" column on the root network page on Taostats](https://taostats.io/subnets/netuid-0/). These percentages for all SNs add up to `100`. 
-    :::
-- At the end of every tempo, i.e., every 360 blocks in a user-created subnet, the TAO ($\tau$) accumulated for each subnet is injected into the subnet. This emitted TAO for the subnet is then allocated among the subnet's miners, validators, and their stakers, as well as the subnet creator.
+### Injection
 
-## Before you proceed
+The first stage of emissions is *injection of liquidity* into the subnet pools. Each block:
 
-Read the [Root Network](./root-network.md) document before you proceed.
+- TAO is injected into the subnet's TAO reserve.
+- alpha is injected into the subnet's alpha reserve.
+- alpha is allocated to *alpha outstanding*, to be extracted by participants.
 
-In the rest of this document we consider the subnet weights (set by the root validators) as **inputs** and proceed to present emission calculations as **outputs**. 
+Liquidity injection to each subnet is in proportion to the price of its token compared to the price other subnet tokens. This is designed to incentivize development on the most valuable subnets. Recall that price is equal to the ratio of the subnet's TAO in reserve to alpha in reserve.
 
-## Subnet weights, trust, rank, consensus, emission 
+#### TAO reserve injection
```

Comment:
should these be capitalized like "TAO Reserve Injection"?

Reply:
nah I don't think so, I could go either way but I don't think they need to be made into special terms or anything, they don't get referred to that often.